src/main/java/net/sf/jabref/gui/help/AboutDialogViewModel.java

```diff
@@ -21,74 +6,54 @@
 import javafx.beans.property.ReadOnlyStringWrapper;
 import javafx.fxml.FXML;
 import javafx.scene.control.Button;
+import javafx.scene.image.Image;
+import javafx.scene.image.ImageView;
 import javafx.stage.Stage;
 
 import net.sf.jabref.Globals;
 import net.sf.jabref.JabRefGUI;
 import net.sf.jabref.gui.ClipBoardManager;
 import net.sf.jabref.gui.desktop.JabRefDesktop;
 import net.sf.jabref.logic.l10n.Localization;
+import net.sf.jabref.logic.util.BuildInfo;
 
+import de.codecentric.centerdevice.javafxsvg.SvgImageLoaderFactory;
 import org.apache.commons.logging.Log;
 import org.apache.commons.logging.LogFactory;
 
 public class AboutDialogViewModel {
 
+    private final String HOMEPAGE = "http://www.jabref.org";
+    private final String DONATION = "http://www.jabref.org/#donations";
+    private final String LIBRARIES = "https://github.com/JabRef/jabref/blob/master/external-libraries.txt";
+    private final String GITHUB = "https://github.com/JabRef/jabref";
+    private final String CHANGELOG = "https://github.com/JabRef/jabref/blob/master/CHANGELOG.md";
```

Comment:
Can we add a deep link here?

I would suggest https://github.com/koppor/keep-a-changelog/blob/master/CHANGELOG.md#unreleased for the development versions and add a release TODO to replace this by something like `https://github.com/JabRef/jabref/blob/master/CHANGELOG.md#36---2016-08-26`